02-configuration.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: page
+title: Version Control with Mercurial
+subtitle: Configuring Mercurial
+---
+> ## Learning Objective {.objectives}
+>
+> * Explain what configuration steps are required the first time Mercurial
+>   is used on a computer.
+
+When we use Mercurial on a computer for the first time,
+we need to configure a few things.
+The command
+
+~~~ {.bash}
+$ hg config --edit
```

Comment:
On a new Ubuntu VM with hg ver 2.8.2 installed, this command did not work (hg: unknown command 'config').  I think in the old hg lesson we just had students editing the config file manually.